include/evmc/evmc.h

```diff
@@ -86,7 +86,8 @@ enum evmc_call_kind
 /** The flags for ::evmc_message. */
 enum evmc_flags
 {
-    EVMC_STATIC = 1 /**< Static call mode. */
+    EVMC_STATIC = 1,   /**< Static call mode. */
+    EVMC_DELEGATED = 2 /**< Delegated call mode (EIP-7702). Valid since Prague. */
```

Comment:
Should we consider "designated" name?

Reply:
What do you mean? "Delegation designation" in the EIP refers to `0xef0100 || address` string in account's code. Here the flag is about execution mode.